Taskfile.yml

```diff
@@ -301,10 +303,13 @@ tasks:
       --set azureClientID=$AZURE_CLIENT_ID \
       --set azureClientSecret=$AZURE_CLIENT_SECRET"
     - "kubectl create namespace pre-release"
-    - "kubectl wait --for=condition=ready --timeout=2m pod -n {{.ASO_NAMESPACE}} -l control-plane=controller-manager"
+    # TODO: Can remove the -c parameter once we GA
```

Comment:
Done, also explained more why we're using it here and thus why it can be removed when we GA.

Specifically:
```
   # TODO: Can remove the -c parameter once we GA. -c means skip waiting for CRDs to reach established state.
   # TODO: this needs to be skipped because the CRD label for filtering doesn't exist in the old beta charts.
```